### ISSUE TITLE
Hi, I tested today and it's works great, thanks.

### ISSUE THREAD

**Issue by nigelsheffield** [kingkong-rc/TINY-TX-X8#4]
Would it be possible option combine both protocols in one firmware and select them with a gesture at switch on?

**Comment by LA2003** [comment]
hi, i will make another D8 FW, but will not make this 3 type protocol together, it's too complex

**Comment by nigelsheffield** [comment]
Ok no problem.
I've now flown a quad with this and a frsky xm plus and it seemed quite difficult to control, there seemed to be some lag making acro mode tricky but level mode was fine. 
Maybe the d8 version will work better.
On my taranis in d8 mode and hall sensor gimbals there is no problem in acro mode, obviously the gimbals themselves will be a major contributor to this ease of flying on the taranis.

I look forward to trying out the d8 version.
Great work 
Thanks.

**Comment by LA2003** [comment]
Hi, refresh rate of D16 about 9ms,  I don't think u can feel delay between the TX and FC, if u fly the D16 like this, D8 will the same. There have expo in middle area of  joystick to avoid the errors when joystick back to centre. This will weakens the TX's  centre area respond, I suggest u make same expo compensation for AIL / ELE / ROL channel in BETAFLIGHT around 1400us~1600us, that will speed up the  centre area respond.

**Comment by nigelsheffield** [comment]
Right, I did wonder if there was some expo settings used in the tx to compensate for gimbal centering errors.
I will try what you suggest then.
But it might just be best to keep this control for level mode in that case.
Thanks again.

**Comment by mijo5124** [comment]
I'd like to figure out how to flash the frsky D16 firmware onto my tiny 8 can someone help me out

**Comment by KaitlynSeU** [comment]
@mijo5124 
Go to this page : https://github.com/kingkong-rc/TINY-TX-X8/releases
Download files below  [FRSKY ACCST D16 non-EU 2018-01-05-BUILD] tab
you need read [README.docx] first  , the D16 firmware in [FRSKY_D16_80105.zip] use .LDAFW extension.  

Flash firmware need use <KINGKONG UPDATE TOOLS.exe>, first time need install X8 radio's driver, the driver files in the bottom row of KINGKONG UPDATE TOOLS's menu, click the <?> icon.
